src/arrays/arrays_with_methods/medium/q9/TwoDAndThatsAFact.java

```diff
@@ -0,0 +1,37 @@
+package arrays.arrays_with_methods.medium.q9;
+
+/*
+Make a program that will calculate the factorial of each element in a 2D array and store the factorial of each element
```

Comment:
I think we should tweak this question text to mention we give an array in the main method and print the output of that generated 2D array.